include/SH3/stdtype.hpp

```diff
@@ -86,6 +86,9 @@ static inline void messagebox(const char* title, const char* str, ...)
 #define SHFALSE 0
 #define SHTRUE  1
 
+#define SH_OK   0
```

Comment:
Should `SH_{OK,BAD}` not follow the same convention as `SH{TRUE,FALSE}` (either with or without underscore)?

Also I think an enum would be good instead of macros.